app/src/main/java/nextstep/jwp/util/Methods.java

```diff
@@ -0,0 +1,20 @@
+package nextstep.jwp.util;
+
+public enum Methods {
+    GET("GET"),
+    POST("POST");
+
+    private final String method;
+
+    Methods(String method) {
+        this.method = method;
+    }
+
+    public static boolean isGet(String targetMethod) {
+        return "GET".equals(targetMethod);
+    }
+
+    public static boolean isPost(String targetMethod) {
+        return "POST".equals(targetMethod);
+    }
```

Comment:
```suggestion
    public static boolean isGet(String targetMethod) {
        return GET.name().equalsIgnoreCase(targetMethod);
    }

    public static boolean isPost(String targetMethod) {
        return POST.name().equalsIgnoreCase(targetMethod);
    }
```

마침 상수화 되어있는 것과 마찬가지이니, 이렇게 코드 스멜을 없애보는 것도...?!